inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -769,6 +769,15 @@ std::shared_ptr<CNNNetworkImpl> convertFunctionToICNNNetwork(const std::shared_p
         cnnLayer->insData.resize(inputCount);
 
         for (size_t i = 0; i < layer->get_output_size(); i++) {
+
```

Comment:
Please, remove empty line and add a comment why is this check needed.

Reply:
Updated.